src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars__container {
+  width: 96px;
+  height: 116px;
+  border-radius: 0.5px;
```

Comment:
Don't fix container size (if there is no such a requirement). Let the content size dictate it to avoid overflow or accidental scrollbar. If this fixed size is not a specific requirement, consider using `min-width` and `min-height` instead or padding to control the size.